keyboards/moondrop/dash75/r1/config.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2023 moondroplab (@moondroplab)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define SERIAL_NUM "Dash 75"
```

Comment:
SERIAL_NUM isnt a thing
```suggestion
```

Reply:
sorry ... typo ... should be `SERIAL_NUMBER`.  it's used in tmk_core as part of the usb descriptor code.  in this brand it's used to distinguish from the embedded DAC
![image](https://github.com/qmk/qmk_firmware/assets/366673/e93e1eb7-3728-41a7-8072-d235cf0b2c57)
*(will populate the "serial Number" blank in the above picture)*